self-hosting/nginx/nginx.conf

```
@@ -0,0 +1,76 @@
+events {}
+
+http {
+    sendfile    on;
+    keepalive_timeout 60;
+
+     upstream omnivore_web {
+    	ip_hash;
+    	server 127.0.0.1:3000;
+     }
+
+     upstream omnivore_backend {
+    	ip_hash;
+    	server 127.0.0.1:4000;
+     }
+
+     upstream omnivore_imageproxy {
+    	ip_hash;
+    	server 127.0.0.1:1010;
+     }
+
+    upstream omnivore_bucket {
+    	ip_hash;
+    	server 127.0.0.1:7070;
+    }
+
+    server {
+        listen 80;
+        return 301 https://$host$request_uri
+    }
+
+    server {
+        listen 443;
+        server_name  omnivore.domain.com;
+
+
+        ssl_certification   /path/to/cert.crt;
+        ssl_certificate_key /path/to/cert.key;
+
+        ssl on;
```

Note sure if this is a version issue or not but on nginx 1.18.0
 - `ssl on` is deprecated in favour of putting it in the `listen` directive
 - `ssl_certification` is invalid and should be `ssl_certificate`

```suggestion
        listen 443 ssl;
        server_name  omnivore.domain.com;


        ssl_certificate   /path/to/cert.crt;
        ssl_certificate_key /path/to/cert.key;
```

---

Thanks for this. I tested it using a modified version with HTTP rather than HTTPS, hence why this was missed